### PR TITLE
add jenkins autoprovision extended tests

### PR DESCRIPTION
**Pull request by gabemontero**
@openshift/sig-developer-experience fyi/ptal

### CODE REVIEW

File: test/extended/builds/jenkins_autoprovision.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		g.It("where jenkins is autoprovisioned by the admission controller", func() {
+
+			// instantiate the bc
+			g.By(fmt.Sprintf("calling oc new-app -f %q", envVarsPipelinePath))
+			err := oc.Run("new-app").Args("-f", envVarsPipelinePath).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// start the build
+			g.By("starting the pipeline build, including env var, and waiting for it to complete")
+			br, err := exutil.StartBuildAndWait(oc, "-e", "FOO2=BAR2", "sample-pipeline-withenvs")
```
Comment (bparees) [inline test/extended/builds/jenkins_autoprovision.go]: we don't need to run anything, it should be sufficient to confirm the jenkins pod gets created.  (don't even really need to wait for it to start up)

Comment (gabemontero) [reply test/extended/builds/jenkins_autoprovision.go]: update pushed

File: test/extended/builds/jenkins_autoprovision.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Builds] jenkins autoprovision", func() {
```
Comment (bparees) [inline test/extended/builds/jenkins_autoprovision.go]: let's be brave and mark it conformance.

Comment (gabemontero) [reply test/extended/builds/jenkins_autoprovision.go]: updated

**Pull request merged**